CommandLine/testing/main.cpp

```diff
@@ -0,0 +1,21 @@
+#include <gtest/gtest.h>
+
+#include <string>
+#include <cstdlib>
+#include <iostream> 
+
+std::string gitMasterDir = "/tmp/enigma-master";
+
+int main(int argc, char **argv) {
+  ::testing::InitGoogleTest(&argc, argv);
+
+  if (argc == 1) {
+    int ret = system(("bash ./ci-regression.sh " + gitMasterDir).c_str());

```

Comment:
You wouldn't need `.c_str()` if you used a macro for that. Just sayin'.
I don't really like how `system()` is spreading through this harness... but that's a bridge we'll burn later.